fictadvisor-api/src/v2/utils/documentation/group/addEmails.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiDocumentationParams } from '../decorators';
+import { ShortUsersResponse } from '@fictadvisor/utils';
+import { DefaultForbiddenResponse, DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const GroupDocumentationAddEmails: ApiDocumentationParams = {
+  isAuth: true,
+  ok: {
+    type: ShortUsersResponse,
+  },
+  badRequest: {
+    description: `\n
+    InvalidGroupIdException: 
+      Group with such id is not found
+      
+    InvalidBodyException: 
+      Email cannot be empty
+      The email is not a valid email address`,
```

Comment:
write valid error messages for min and max array size in EmailDTO, add `@IsNotEmpty()`